tts/UI.def.lua

```diff
@@ -96,12 +98,17 @@ UI = {}
 ---@field attributes nil | tts__UIElementBase_Attributes
 ---@field children nil | tts__UIElement[]
 
----@shape tts__UIInputElementBase : tts__UIElementBase
+---@shape tts__UIInputElementBase_Attributes: tts__UIElementBase_Attributes
```

Comment:
`---@shape Child : Parent` is preferred i.e. spaces surrounding `:`.

Reply:
Done